blocks/video-player/video-player.css

```diff
@@ -0,0 +1,52 @@
+/* video-player specific style */
+
+/* hide first child as it only contains the video url */
+.video-player.block > div:first-child {
+    display: none;
+}
+
+.video-player-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.9;
+    background: #000;
+    z-index: 100;
+}
+
+.video-player-iframe {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 103;
+    display: block;
+    width: 1280px;
+    max-width: 90%;
+    max-height: 90%;
+    aspect-ratio: 1280/720;
+    border: none;
+    box-shadow: 0 0 15px hsl(0deg 0% 0% / 80%);
+}
+
+.video-player-toolbar {
+    display: flex;
+    justify-content: center;
+    position: fixed;
+    top: 11px;
+    left: 10px;
+    z-index: 110;
+    width: 25px;
+    height: 23px;
+    background: #000;
+}
+
+.video-player-close {
+    content: url("../../icons/close.svg");
```

Comment:
if this is not re-used by other blocks, you can also move this under `/blocks/video-player`